Formula/principalmapper.rb

```diff
@@ -0,0 +1,55 @@
+class Principalmapper < Formula
+  include Language::Python::Virtualenv
+
+  desc "Quickly evaluate IAM permissions in AWS"
+  homepage "https://github.com/nccgroup/PMapper"
+  url "https://files.pythonhosted.org/packages/b4/4d/c270b88e05d4b17705936b7a3d05056d67e117b4087c663834d9bbd348d6/principalmapper-1.1.2.tar.gz"
+  sha256 "e36366a603b812de9eaf9c242a421f6982517b236344b9c44a1c509d82c62a26"
+  license "AGPL-3.0-only"
```

Comment:
This seems to indicate that it's AGPL-3.0-or-later:
https://github.com/nccgroup/PMapper/blob/6139b13e4a91cb17a5817e7127de9f3bd939eeda/principalmapper/__init__.py#L5-L8